digits/model/tasks/torch_train.py

```diff
@@ -250,7 +250,10 @@ def task_arguments(self, resources, env):
             args.append('--type=float')
 
         if self.pretrained_model:
-            args.append('--weights=%s' % self.path(self.pretrained_model))
+            filenames = self.pretrained_model.split(os.path.pathsep)
+            if len(filenames) > 1:
+                raise ValueError('Torch does not support multiple pretrained model files')
+            args.append('--weights=%s' % self.path(filenames[0]))
```

Comment:
that looks good though (probably not due to this) it seems the error came too quickly to show in the UI, as if we hadn't yet created the socketIO interface. The error message only showed after I reloaded the page.
I tested that it works when only one pretrained model is provided.